File: Backend/application_service/core/domain/delegation_member.go
package domain

import "github.com/google/uuid"

type DelegationMember struct {
	ID                   uuid.UUID
	FullName             string
	Email                string
	Gender               Gender
	PositionID           uuid.UUID
	Position             DelegationMemberPosition
	Image                string
	SportsOrganizationID uuid.UUID
	SportsOrganization   SportsOrganization
}

File: Frontend/Private/gco-private/src/app/model/core/delegation-member.ts
import { DelegationMemberPosition } from "./delegation-member-position";
import { Gender } from "./gender";
import { SportsOrg } from "./sports-org";

export interface DelegationMember {
    id?: string;
    fullName: string;
    email: string;
    gender:Gender; 
    position: DelegationMemberPosition;
    sportsOrganisation? : SportsOrg;
} 
